app/client/components/cancel/voucher/voucherCancellationFlowStart.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import { getMainPlan, Subscription } from "../../../../shared/productResponse";
+import { trackEvent } from "../../analytics";
+import { PageContainerSection } from "../../page";
+import { hrefStyle } from "../cancellationConstants";
+
+const trackCancellationClickEvent = (eventLabel: string) => () =>
+  trackEvent({
+    eventCategory: "cancellation",
+    eventAction: "click",
+    eventLabel
+  });
+
+export const voucherCancellationFlowStart = (subscription: Subscription) => {
+  const mainPlan = getMainPlan(subscription);
+
+  const isEligibleForFreeDigipackAccess =
+    mainPlan?.name?.indexOf("plus Digi") === -1;
```

Comment:
`!mainPlan?.name.includes("plus Digi")`

Reply:
wasn't sure if `includes` has babel issues

Reply:
you're correct, ignore this. Babel transpiles ES6 code but doesn't polyfill ES6 methods